src/agent/coverage/src/block.rs

```diff
@@ -251,37 +254,45 @@ mod tests {
         );
     }
 
-    fn module_path(path: &str) -> ModulePath {
-        ModulePath::new(std::path::PathBuf::from(path)).unwrap()
+    // Given a POSIX-style path as a string, construct a valid absolute path for
+    // the target OS and return it as a checked `ModulePath`.
+    macro_rules! module {
```

Comment:
is there an advantage of using a module instead of a function ?

Reply:
Not anymore (other than the use site standing out a bit more). I was originally going to make it accept an input spec in a different format, then ended up going with this simpler / more function-friendly impl.